go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/kr/pretty v0.1.0 // indirect
 	github.com/spf13/afero v1.2.2 // indirect
 	github.com/stretchr/testify v1.4.0
+	github.com/taskcluster/httpbackoff v1.0.0
```

Comment:
This ends up still using v0.  I think the fix is to add `/v3` to imports of this package.

Reply:
I did that -- r?